winfsp/src/vsb.rs

```diff
@@ -54,6 +54,11 @@ impl<T> VariableSizedBox<T> {
     pub fn len(&self) -> usize {
         self.size
     }
+
+    /// Returns `true` if it contains no elements.
+    pub fn is_empty(&self) -> bool {
```

Comment:
This isn't used anywhere and doesn't make sense for the semantics of `VariableSizedBox`